arborio/cv_policy_parse.cpp

```diff
@@ -0,0 +1,218 @@
+#include <any>
+#include <limits>
+#include <optional>
+
+#include <arborio/label_parse.hpp>
+#include <arborio/cv_policy_parse.hpp>
+
+#include <arbor/arbexcept.hpp>
+#include <arbor/cv_policy.hpp>
+#include <arbor/s_expr.hpp>
+#include <arbor/util/expected.hpp>
+
+
+#include "parse_helpers.hpp"
+
+namespace arborio {
+namespace {
+
+template<typename T> using parse_hopefully = arb::util::expected<T, cv_policy_parse_error>;
+
+std::unordered_multimap<std::string, evaluator>
+eval_map {{"default",
+           make_call<>([] () { return arb::cv_policy{arb::default_cv_policy()}; },
+                       "'default' with no arguments")},
+          {"every-segment",
+           make_call<>([] () { return arb::cv_policy{arb::cv_policy_every_segment()}; },
+                       "'every-segment' with no arguments")},
+          {"every-segment",
+           make_call<region>([] (const region& r) { return arb::cv_policy{arb::cv_policy_every_segment(r) }; },
+                             "'every-segment' with one argument (every-segment <region-expr>)")},
```

Comment:
Small nitpick: in the other parsers, the style of the message is like this: 
```
'every-segment' with one argument (reg:region)
```
If it's not too much work can we unify the representation?